tests/test_tensor/test_zero_optim.py

```diff
@@ -6,7 +6,7 @@
 from colossalai.utils.cuda import get_current_device
```

Comment:
What is the difference between this file and `test_optim.py` in `test_gemini/update`?

Reply:
It test tp, dp hybrid parallelism.

Reply:
Can you rename it to make its functionality clear?

Reply:
ok.